collectors/bzimport/tests/test_convertors.py

```diff
@@ -361,6 +393,107 @@ def test_meta_removed(self):
         assert flaw.meta.first() == self.get_meta(flaw)[1]
         assert flaw.meta.first().meta_attr["name"] == "Lone Wanderer"
 
+    def test_acknowledgment_removed(self):
```

Comment:
Could you please use `@pytest.mark.parametrize` for these tests?

Reply:
I'm afraid not.

Would the fixture scope be clear then? I'm afraid setting the fixture scope so that they survive the individual parametrized calls would in the end be more complicated than what is there now.

Reply:
You do not need to deal with a scope. Create a flaw,  perform an operation on it and then check the result.

Reply:
Then the `*_orig` variables would overreach the scope of each parametrized test. A workaround for that would be more complex than the current test. And it would change the style of the test from the rest. Feel free to submit a follow-up MR.